package.json

```diff
@@ -56,6 +56,7 @@
     "@size-limit/preset-big-lib": "^5.0.3",
     "@types/bun": "^1.0.6",
     "@types/node": "^12.0.2",
+    "@types/jest": "^29.5.12",
```

Comment:
**issue:** we don't actually use Jest. We're using [bun test](https://bun.sh/docs/cli/test), which is mostly drop-in compatible with Jest, and the Bun runner will sort of just smooth over missing imports at runtime.

But to fix the TS error you get with `bun run build`, instead we should add imports to the failing files like `__tests__/core/1525.test.ts` with:

```ts
import { describe, it } from "bun:test"
````

I'll take care of that since it's pretty straightforward and I'm looking at it now. It's awesome that we're type checking the tests now, but we can get it to work from Bun rather than re-importing Jest dependencies. Will push that up shortly.

Reply:
Updated with b687398d2cbb42a536d5a309ba3d7b9b6031c5b4, and I found a few places where we needed `expect.fail()` instead of `fail()`. `bun run build` and `bun run test:all` still works on that commit.